calcite-adapter/src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbRules.java

```diff
@@ -319,8 +320,14 @@ private void initializeRexCallToMongoMap(final Instant currentTime) {
             rexCallToMongoMap.put(SqlStdOperatorTable.CASE, RexToMongoTranslator::getMongoAggregateForCase);
             rexCallToMongoMap.put(SqlStdOperatorTable.ITEM, RexToMongoTranslator::getMongoAggregateForItem);
 
-            rexCallToMongoMap.put(SqlStdOperatorTable.SUBSTRING,
-                    RexToMongoTranslator::getMongoAggregateForSubstringOperator);
+            // String operations
+            rexCallToMongoMap.put(SqlStdOperatorTable.CONCAT, StringFunctionTranslator::getMongoAggregateForConcatOperator);
```

Comment:
CONCAT is the binary string concat operator  - string || string  - This one returns null if either string is null.
CONCAT_FUNCTION takes two or more strings - CONCAT(string, string, string) - This one converts any null string to empty string.